tgui/packages/tgui/components/DepartmentPane.tsx

```diff
@@ -0,0 +1,73 @@
+import { Color } from 'common/color';
+import { SFC } from 'inferno';
+import { Box } from './Box';
+import { Stack } from './Stack';
+
+// The cost of flexibility and prettiness.
+export const DepartmentEntry: SFC<{
+  style?;
+  titleStyle?;
+  textStyle?;
+  title?;
+  titleContents?;
+  titleSubtext?;
+}> = (props) => {
+  return (
+    <Box ClassName="Section" style={props.style}>
```

Comment:
Also why are we redefining Section?

Reply:
Not actually, see latest commit that took a bit to push cause I was being picky with colors yet again